<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenPassword</class>
 <widget class="QDialog" name="GenPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>生成随机密码</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src/icons.qrc">
    <normaloff>:/mainui/icon/MainIcon.ico</normaloff>:/mainui/icon/MainIcon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>80</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QCheckBox" name="cbx_uppercase">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox {
    padding: 2px;margin: 2px;
}
QCheckBox::indicator {
    width: 15px;
    height: 15px;
}
QCheckBox::indicator:unchecked {
    border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#34495E;
  	background-color:rgb(255,255,255);
}
QCheckBox::indicator:checked {
	border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#0078d7; 
	image: url(:/mainui/icon/selected.png);
}
QCheckBox::indicator:hover{
	border-color:#0078d7; 
}
QCheckBox:hover {
	color:rgb(0, 0, 216);
	padding: 3px;
}</string>
         </property>
         <property name="text">
          <string>大写字母</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbx_lowercase">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox {
    padding: 2px;margin: 2px;
}
QCheckBox::indicator {
    width: 15px;
    height: 15px;
}
QCheckBox::indicator:unchecked {
    border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#34495E;
  	background-color:rgb(255,255,255);
}
QCheckBox::indicator:checked {
	border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#0078d7; 
	image: url(:/mainui/icon/selected.png);
}
QCheckBox::indicator:hover{
	border-color:#0078d7; 
}
QCheckBox:hover {
	color:rgb(0, 0, 216);
	padding: 3px;
}</string>
         </property>
         <property name="text">
          <string>小写字母</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbx_digital">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox {
    padding: 2px;margin: 2px;
}
QCheckBox::indicator {
    width: 15px;
    height: 15px;
}
QCheckBox::indicator:unchecked {
    border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#34495E;
  	background-color:rgb(255,255,255);
}
QCheckBox::indicator:checked {
	border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#0078d7; 
	image: url(:/mainui/icon/selected.png);
}
QCheckBox::indicator:hover{
	border-color:#0078d7; 
}
QCheckBox:hover {
	color:rgb(0, 0, 216);
	padding: 3px;
}</string>
         </property>
         <property name="text">
          <string>数字</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbx_char">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox {
    padding: 2px;margin: 2px;
}
QCheckBox::indicator {
    width: 15px;
    height: 15px;
}
QCheckBox::indicator:unchecked {
    border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#34495E;
  	background-color:rgb(255,255,255);
}
QCheckBox::indicator:checked {
	border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#0078d7; 
	image: url(:/mainui/icon/selected.png);
}
QCheckBox::indicator:hover{
	border-color:#0078d7; 
}
QCheckBox:hover {
	color:rgb(0, 0, 216);
	padding: 3px;
}</string>
         </property>
         <property name="text">
          <string>特殊字符</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>最大长度</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>最小长度</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spb_minl">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox {
	border-style:solid;
	border-width:2px;
	border-radius:6px;
	border-color:#34495E;
	padding:1px;
	margin: 1px;
}
QSpinBox:hover{
	color:rgb(0, 0, 216);
	border-color:#0078d7;
}
QSpinBox::up-button {
    padding:1px;
    background-color:transparent;
}
QSpinBox::down-button {
    padding:1px;
    background-color:transparent;
}
QSpinBox::down-button {
    padding:1px;
    background-color:transparent;
}
/* 向上箭头*/
QSpinBox::up-arrow {
	image:url(:/mainui/icon/up1.png);
	height: 15px;
}
QSpinBox::up-arrow:hover {
	
	image: url(:/mainui/icon/up0.png);
	height: 15px;
}
QSpinBox::up-arrow:pressed {
	
	image: url(:/mainui/icon/up0.png);
	height: 13px;
}
/*向下箭头*/
QSpinBox::down-arrow {
	image: url(:/mainui/icon/down1.png);
	height: 15px;
}
QSpinBox::down-arrow:hover {
	image: url(:/mainui/icon/down0.png);
	height: 15px;
}
QSpinBox::down-arrow:pressed {
	image: url(:/mainui/icon/down0.png);
	height: 13px;
}
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spb_maxl">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox {
	border-style:solid;
	border-width:2px;
	border-radius:6px;
	border-color:#34495E;
	padding:1px;
	margin: 1px;
}
QSpinBox:hover{
	color:rgb(0, 0, 216);
	border-color:#0078d7;
}
QSpinBox::up-button {
    padding:1px;
    background-color:transparent;
}
QSpinBox::down-button {
    padding:1px;
    background-color:transparent;
}
QSpinBox::down-button {
    padding:1px;
    background-color:transparent;
}
/* 向上箭头*/
QSpinBox::up-arrow {
	image:url(:/mainui/icon/up1.png);
	height: 15px;
}
QSpinBox::up-arrow:hover {
	
	image: url(:/mainui/icon/up0.png);
	height: 15px;
}
QSpinBox::up-arrow:pressed {
	
	image: url(:/mainui/icon/up0.png);
	height: 13px;
}
/*向下箭头*/
QSpinBox::down-arrow {
	image: url(:/mainui/icon/down1.png);
	height: 15px;
}
QSpinBox::down-arrow:hover {
	image: url(:/mainui/icon/down0.png);
	height: 15px;
}
QSpinBox::down-arrow:pressed {
	image: url(:/mainui/icon/down0.png);
	height: 13px;
}
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="tb_brower">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextBrowser {
	border-style:none;
	padding:3px;
	border-radius:8px;
	border:2px solid #DCE4EC;
}
QTextBrowser:hover,QTextBrowser:focus {
	border:2px solid #34495E;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbt_gen">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>生成密码</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-style:none;
    padding:8px;
    border-radius:15px;
    color:#FFFFFF;
    background:#34495E;
}
QPushButton:hover {
    color:#4E6D8C;
    background:#e5e5e5;
	border:2px solid #34495E;
}
QPushButton:pressed {
    color:#000000;
    background:#B8C6D1;
}
QPushButton:disabled{
    background:rgb(134, 134, 134);
    border-color:rgb(134, 134, 134);
}
</string>
         </property>
         <property name="text">
          <string>生成</string>
         </property>
         <property name="shortcut">
          <string>Enter</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbt_sure">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>确认填写</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-style:none;
    padding:8px;
    border-radius:15px;
    color:#FFFFFF;
    background:#34495E;
}
QPushButton:hover {
    color:#4E6D8C;
    background:#e5e5e5;
	border:2px solid #34495E;
}
QPushButton:pressed {
    color:#000000;
    background:#B8C6D1;
}
QPushButton:disabled{
    background:rgb(134, 134, 134);
    border-color:rgb(134, 134, 134);
}</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src/icons.qrc"/>
 </resources>
 <connections/>
</ui>
